.gitignore

```diff
@@ -5,7 +5,6 @@ __pycache__/
 
 # C extensions
 *.so
-
```

Comment:
To keep the diff small, please undo this :)

Reply:
ping!